scapy/contrib/automotive/uds_scan.py

```diff
@@ -574,6 +771,12 @@ class UDS_RCEnumerator(UDS_Enumerator):
     def _get_initial_requests(self, **kwargs):
         # type: (Any) -> Iterable[Packet]
         scan_range = kwargs.pop("scan_range", range(0x10000))
+        rc_start = kwargs.get("rc_start")
```

Comment:
This code should be part of the connector. I suggest to add another argument like "detailed_scan_range" which contains a list of tuples `(routineControlType, routineIdentifier)`.